### DISCUSSION TITLE
Behavior of Aria Attributes in the Latest Version of @react-aria/slider

### DISCUSSION THREAD

**Discussion by elianarlivingston** [adobe/react-spectrum#5098] [Q&A] [answered]
Hello! I hope you are doing well. I would like to inquire about the latest update of @react-aria/slider. I've noticed that with the latest update, my snapshots are automatically updated, and the aria-describedby and aria-details attributes are added with an empty string value. Is this correct? Shouldn't the attributes be removed if they have empty values? I would like to know if this addition was intentional or if there is a specific reason behind it.

![Captura de pantalla 2023-09-08 a la(s) 16 16 36](https://github.com/adobe/react-spectrum/assets/59533564/6914303a-44a7-4450-b96e-e424708ee359)

**Comment by snowystinger** [comment]
Can you provide a reproduction? Either in our docs on in a codesandbox? Otherwise, this can sometimes happen if you install multiple copies of our packages. This is fairly common as the package managers are really more focused on node, not browser js. You could try deleting your node modules and package lock and see if that resolves the issue. Or you can use tools like yarn dedupe if it is duplicate copies of our packages.

**Reply by elianarlivingston** [reply]
Hello, Robert! Yes, I'm sharing this [demo](https://stackblitz.com/edit/vitejs-vite-pf7jld?file=src%2FComponent.jsx) with the example code they have on their website. :)

**Reply by snowystinger** [reply]
Ah, yeah, I see it in our docs as well https://react-spectrum.adobe.com/react-aria/useSlider.html#single-thumb
Thanks for the extra info

It looks like it broke in this PR https://github.com/adobe/react-spectrum/pull/4960

I'll add an issue

**Reply by snowystinger** [reply, marked as answer]
Appears to have been fixed at some point https://github.com/adobe/react-spectrum/issues/5103
